ccm_web/client/src/components/ConfirmationTable.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useState } from 'react'
+import { IconButton, makeStyles, Paper, Table, TableBody, TableCell, TableContainer, TableFooter, TableHead, TablePagination, TableRow, useTheme } from '@material-ui/core'
+import { KeyboardArrowRight, KeyboardArrowLeft, LastPage as LastPageIcon, FirstPage as FirstPageIcon } from '@material-ui/icons'
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexShrink: 0,
+    marginLeft: theme.spacing(2.5)
+  }
+}))
+
+interface StudentGrade {
+  rowNumber: number
+  uniqname: string
+  grade: string
+
+}
+
+interface ConfirmationTableProps {
+  grades: StudentGrade[]
+}
+
+interface TableHeaderColumnInfoShouldUseMatUIType {
+  id: keyof StudentGrade
+  label: string
+  minWidth: number
+  align?: 'left' | 'right' | undefined
+}
+
+const columns: TableHeaderColumnInfoShouldUseMatUIType[] = [
+  { id: 'rowNumber', label: 'Row Number', minWidth: 25 },
+  { id: 'uniqname', label: 'uniqname', minWidth: 100 },
+  { id: 'grade', label: 'grade', minWidth: 100 }
+]
```

Comment:
I don't think we need to do this now, but I'm starting to wonder how we can re-use as much of these table components as possible with other entities (I'm thinking specifically about sections and groups). Have you given any thought to that? I'm thinking about generalizing the props here, also passing the column info, and validation info (or a function)? Generics could help us keep the types consistent: https://www.typescriptlang.org/docs/handbook/2/generics.html If you like the idea, we could make this a separate issue.

Reply:
I thought about this and it seemed like yes, you could abstract this out of ConfirmationTable and ValidationErrorTable but you'd probably end up with that same amount of table specific code in the GradebookCanvas component, so it didn't seem like a clear benefit to me.  As I found the table code somewhat  difficult to read my decision was to 'simplify' it by making 2 separate tables, and maybe come back to it later if appropriate.

Reply:
Seems reasonable. See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/65#discussion_r627755056 for some related commentary. IMO, we don't need to create a new issue now, we can do that organically if we decide later there's stuff we want to re-use.

Reply:
@chrisrrowland can you change the Table header name with capital like `Uniqname` and `Grade`? Currently this is how it looks
![Screen Shot 2021-05-07 at 2 32 19 PM](https://user-images.githubusercontent.com/8579775/117493502-0e256980-af41-11eb-8f98-e5ee6258cf95.png)